draft-irtf-cfrg-frost.md

```diff
@@ -366,21 +369,19 @@ particular input `x`, i.e., `y = f(x)` using Horner's method.
 
   def polynomial_evaluate(x, coeffs):
     value = 0
-    for (counter, coeff) in coeffs.reverse():
-      if counter == coeffs.len() - 1:
-        value += coeff // add the constant term
-      else:
-        value += coeff
-        value *= x
+    for coeff in reverse(coeffs):
+      value *= x
+      value += coeff
     return value
```

Comment:
Uh, doesn't this mean the final constant gets multiplied by x, when it shouldn't? // @chelseakomlo